lib/conversation/replies.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+/* eslint-disable no-param-reassign */
+
+const logger = require('winston');
+
+const Command = require('./command');
+const actions = require('./actions');
+const helpers = require('../helpers');
+const stathat = require('../stathat');
+const BotRequest = require('../../app/models/BotRequest');
+
+function customMsgDecorator(fn) {
+  return (args) => {
+    fn(args);
+    return helpers.sendResponse(args.res, 200, args.text);
+  };
+}
+
+function renderEndConversationMsg(fn) {
+  return (type, args) => helpers.renderMessageForMessageType(args.req, type)
+      .then((replyMessage) => {
+        args.text = replyMessage;
```

Comment:
Similar comment / concern about readability here --  What do you think about renaming this to `args.replyText`?  Because `req` and `res` get both passed as `args`, the general `text` property was throwing me off a little bit. Since we're passing both `req` and `res`  - is there a reason for defining `text` property in `args` vs adding it to the `req` ?